buildpacks/nodejs-engine/src/main.rs

```diff
@@ -1,8 +1,3 @@
-#![warn(unused_crate_dependencies)]
-#![warn(clippy::pedantic)]
-#![warn(clippy::cargo)]
```

Comment:
Intentionally removing the `clippy::cargo` lint, since:
- it only really benefits crates that are being published (and this crate isn't being published)
- it contains some lints that are non-deterministic/potentially noisy (eg https://rust-lang.github.io/rust-clippy/master/index.html#/multiple_crate_versions)